src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -578,7 +578,16 @@
 examples:
   - name: Add bg extension.
     text: >
-        az sf managed-node-type vm-extension add -g testRG -c testCluster -n snt --extension-name csetest --publisher Microsoft.Compute --extension-type BGInfo --type-handler-version 2.1 --auto-upgrade-minor-version
+        az sf managed-node-type vm-extension add -g testRG -c testCluster -n snt --extension-name  MyCustomScriptExtension --publisher Microsoft.Compute --extension-type CustomScriptExtension --type-handler-version 1.10 --auto-upgrade-minor-version --setup-order BeforeSFRuntime --provision-after-extension ServiceFabricMCNodeVmExt
```

Comment:
remove "--provision-after-extension ServiceFabricMCNodeVmExt". Customers should not specify SFMC extensions it is hidden to them. Or a add a different extension name like "--provision-after-extension Microsoft.Azure.Geneva.GenevaMonitoring"